config/formats.ts

```diff
@@ -3285,6 +3311,23 @@ export const Formats: FormatList = [
 			'Exeggutor + Stun Spore + Stomp', 'Jolteon + Focus Energy + Thunder Shock', 'Flareon + Focus Energy + Ember',
 		],
 	},
+	{
+        	name: "[Gen 1] Nintendo Cup 1997",
+        	threads: [
+            	`&bullet; <a href="https://www.smogon.com/forums/threads/3572352/">RBY OU Viability Ranking</a>`,
+            	`&bullet; <a href="https://www.smogon.com/forums/threads/3650478/#post-8133786">RBY Sample Teams</a>`,
+        	],
+
+        	mod: 'gen1jpn',
+        	cupLevelLimit: [50, 55, 155],
+        	teamLength: {
+            		validate: [3, 6],
+            		battle: 3,
+        	},
+        	searchShow: false,
+        	ruleset: ['Obtainable', 'Team Preview', 'Cup Level Limit', 'Sleep Clause Mod', 'Species Clause', 'Nickname Clause', 'HP Percentage Mod', 'Cancel Mod'],
+        	banlist: ['Uber'],
+    	},
```

Comment:
```suggestion
		name: "[Gen 1] Nintendo Cup 1997",
		threads: [
			`&bullet; <a href="https://www.smogon.com/forums/threads/3572352/">RBY OU Viability Ranking</a>`,
			`&bullet; <a href="https://www.smogon.com/forums/threads/3650478/#post-8133786">RBY Sample Teams</a>`,
		],

		mod: 'gen1jpn',
		cupLevelLimit: [50, 55, 155],
		teamLength: {
			validate: [3, 6],
			battle: 3,
		},
		searchShow: false,
		ruleset: ['Obtainable', 'Team Preview', 'Cup Level Limit', 'Sleep Clause Mod', 'Species Clause', 'Nickname Clause', 'HP Percentage Mod', 'Cancel Mod'],
		banlist: ['Uber'],
	},
```
we use tabs to indent, not spaces